FirebaseFunctions/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # v9.0.0
+- [changed] The FirebaseFunctionsSwift library has been removed. All of its APIs are now included
+  in the FirebaseFunctions. Please remove references to FirebaseFunctionsSwift from Podfile's and
+  Swift Package Manager configurations.
```

Comment:
Also need to mention from your import statements (replacing it with `FirebaseFunctions`)